spec/tasks/exec_spec.cr

```diff
@@ -3,7 +3,7 @@ require "../spec_helper"
 describe Lucky::Exec do
   it "runs the editor" do
     with_test_template do
-      Lucky::Exec.new.call(["-e", %(echo '5 + 5' >)])
+      Lucky::Exec.new.call(["-o", "-e", %(echo '5 + 5' >)])
```

Comment:
Just to make these options clearer, could you switch these to the full args?

```suggestion
      Lucky::Exec.new.call(["--once", "--ediitor", %(echo '5 + 5' >)])
```

Reply:
Yep, that's better. I didn't use the suggested change due to a typo, but the update has been made.